ynput/ayon-tvpaint
# Rename TVPaint plugin to AYON plugin

## Changelog Description
TVPaint plugin is called AYONPlugin and is looking for different environment variable for server url.

## Additional review information
I used different env variable so it does not clash with OpenPype plugin which people might still use, in that case both plugins would catch the url, leading to doubling work.

Also made smaller changes in the plugin. It does create 2 requesters, one for plugin menu, second for ticks. So closing AYON menu does not make integration to stop work. And few other smaller changes.

## Testing notes:
1. Create package, upload and use in bundle.
2. Launch TVPaint.
3. Integration should work, and you should see `AYON` in plugins.

Resolves https://github.com/ynput/ayon-tvpaint/issues/17
